src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsCreatePage.tsx

```diff
@@ -0,0 +1,53 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import Navigation from '@libs/Navigation/Navigation';
+import type {FullScreenNavigatorParamList} from '@libs/Navigation/types';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+type WorkspaceWorkflowsApprovalsCreatePageProps = WithPolicyAndFullscreenLoadingProps &
+    StackScreenProps<FullScreenNavigatorParamList, typeof SCREENS.WORKSPACE.WORKFLOWS_APPROVALS_EXPENSES_FROM>;
+
+function WorkspaceWorkflowsApprovalsCreatePage({policy, isLoadingReportData = true, route}: WorkspaceWorkflowsApprovalsCreatePageProps) {
+    const {translate} = useLocalize();
+
+    // eslint-disable-next-line rulesdir/no-negated-variables
+    const shouldShowNotFoundView = (isEmptyObject(policy) && !isLoadingReportData) || !PolicyUtils.isPolicyAdmin(policy) || PolicyUtils.isPendingDeletePolicy(policy);
+
+    return (
+        <AccessOrNotFoundWrapper
+            policyID={route.params.policyID}
```

Comment:
```suggestion
            policyID={route?.params?.policyID ?? '-1'}
```

Reply:
Why do you suggest that @DylanDylann? `route.params.policyID` is a string, so I don't think we need optional chaining here. I just added a [small fix](https://github.com/Expensify/App/pull/45995/commits/40eb51b2cc8f561157decce7422d0005f97ed24e), maybe this was causing your IDE to show an error?